sdk/component/internal/storer/storer.go

```diff
@@ -0,0 +1,5 @@
+package storer
+
+import "errors"
+
+var ErrNoFindingsFound = errors.New("no findings found")
```

Comment:
This lives here so it's not tightly coupled to the local db but can be swapped with the remote one.